advanced/configuration.md

```diff
@@ -1,4 +1,69 @@
 # Configuration
 
+Mimic is a fully-configurable app. It allows you to configure a JSON file with the port settings and your projects' details.
 
+The file is called `apimocker.json` and you can find it in the app package content. The config has a JSON structure.
 
+Example Config:
+
+```
+{
+  "httpPort": "3000",
+  "httpsPort": "3001",
```

Comment:
We can remove this field entirely since we're not going to implement SSL in the end.